### DISCUSSION TITLE
Обновление для работы с новым интерфейсом.

### DISCUSSION THREAD

**Discussion by Gute4Nacht** [Chimildic/YaMuTools#105] [Идея]
Прошу обновить программу для работы с новым интерфейсом Яндекс музыки, т.к. лично у меня в нем не появляется кнопка управления (что ясно, ведь поменялся даже не сам сайт, а даже ссылка, по которой он располагается). Очень хотелось бы прекратить переключаться между старой версией и новой для сортировки и пользования соответственно.

**Comment by Chimildic** [comment]
Боюсь что просто обновления адреса недостаточно. Поменялся не только дизайн. Изменились механизмы работы сайта и API. По сути нужно писать расширение с нуля. Как в части интерфейса, так и в части алгоритмов. Достаточно большая работа. 
Как вы понимаете, разработчик только я и пока нет желания заниматься этим.

Поэтому как только Яндекс отключит старый дизайн, нынешняя версия расширения станет бесполезной.

**Reply by Leonid-glitch** [reply]
а есть какой-то вариант вернуть старый дизайн?

**Reply by EldarAliev** [reply]
Да.
 Нажать на свой аватар (в левом нижнем углу и в меню выбрать "К старому дизайну".
